asteroid/utils/hub_utils.py

```diff
@@ -190,7 +190,7 @@ def hf_get_from_cache(
     resume_download=False,
     user_agent: Union[Dict, str, None] = None,
     local_files_only=False,
-) -> Optional[str]:
+) -> Optional[str]:  # pragma: no cover
```

Comment:
:joy: :joy: :joy:

Reply:
Let's assume this is tested in HF, we'll test it later if we have issues with it.